File: Taller del preparcial/Punto3.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
    double Numero_decimal;

    cout << "Introduce un numero decimal: ";
    cin >> Numero_decimal; 

    double Valor_a = fabs(Numero_decimal);

    if (Numero_decimal == 0) {
        cout << "0";
    }
    else if (Numero_decimal > 0) {
        cout << "El número es positivo" << endl;
        if (Valor_a < 1){
            cout << " y es pequeño";
        }
        else if (Valor_a > 1000000){
                cout << " y es grande";
        } 
    }
    else{
        cout << "El número es negativo" << endl;
        if (Valor_a < 1){
            cout << " y es pequeño";}
        else if (Valor_a > 1000000){
            cout << " y es grande";}
    }

    
}

File: Taller del preparcial/Punto3.py
Numero_decimal= float(input("Introduce un número decimal: "))

Valor_a = abs(Numero_decimal)

if Numero_decimal == 0:
    print (0)
elif Numero_decimal > 0 :
    print("El número es positivo",end="")
    if Valor_a < 1:
        print(" y pequeño")
    elif Valor_a > 1000000:
        print(" y grande")
    else:
        print()
else: 
    print("El número es negativo",end="")
    if Valor_a < 1:
        print(" y pequeño")
    elif Valor_a > 1000000:
        print(" y grande")
    else:
        print()

